src/operator/mkl/mkl_elementwise_copy-inl.h

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
```

Comment:
have new implementation of element wise , but will waiting for next mklml release to full support it